docs/upgrade_guide.md

```diff
@@ -0,0 +1,30 @@
+# Upgrade Guide
+
+This guide describes how to upgrade Open Service Mesh (OSM).
+
+## Prerequisites
+- Kubernetes cluster with OSM installed
+- The [helm 3 CLI](https://helm.sh/docs/intro/install/)
+
+## Upgrade OSM using Helm
+Use the `helm` CLI to upgrade the OSM control plane from a Kubernetes cluster.
+
+Run `helm upgrade`. 
+```console
+# Add steps for helm upgrade
+```
+
+Run `helm upgrade --help` for more options.
+
+## CRD Upgrades
+Please check the `CRD Updates` section of the [release notes](https://github.com/openservicemesh/osm/releases) to see if additional steps are required to update the CRDs used by OSM. If the new release does contain updates to the CRDs, it is required to first delete existing CRDs and the associated Custom Resources prior to upgrading.
```

Comment:
👍 
May want to also add that the end user will also have to recreate the CRDs and CRs once the control plane upgrade is complete